src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
```

Comment:
The `alt` attribute for your image should be more descriptive. It helps with accessibility and SEO. For example, it could be 'Moyo company logo' instead of just 'Moyo logo'.